features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml

```diff
@@ -561,6 +562,57 @@
         </table>]]></body>
         <footer>---</footer>
     </configuration>
+    <configuration type="EmailOTP" display="EmailOTP" locale="en_US" emailContentType="text/html">
+        <subject>WSO2 - Email OTP</subject>
```

Comment:
Shall we add some more context to the subject? WDYT?

Reply:
Added.